src/guides/v2.4/install-gde/prereq/php-settings.md

```diff
@@ -6,7 +6,7 @@ functional_areas:
   - System
   - Setup
 redirect_from: 
-  - /guides/v2.3/install-gde/prereq/php-settings-ubuntu.html
+  - /guides/v2.4/install-gde/prereq/php-settings-ubuntu.html
```

Comment:
Remove this redirect. The file never existed in 2.4.

Reply:
Fixed